sdk/appconfiguration/app-configuration/review/app-configuration.api.md

```diff
@@ -43,7 +43,7 @@ export interface ClearReadOnlyResponse extends ConfigurationSetting, SyncTokenHe
 
 // @public
 export interface ConfigurationSetting extends ConfigurationSettingParam {
-    lastModified?: Date;
+    lastModifiedOn?: Date;
```

Comment:
Oh interesting, so this was decided in an API review? Storage-blob uses `lastModified` in a lot of its responses.

Reply:
Oof, I went back and listened to the recording - my memory was faulty on that one. It should stay as lastModified. I'll scrub that.